test/e2e/utils/daemonset/daemonset.go

```diff
@@ -0,0 +1,42 @@
+package daemonset
```

Comment:
Even if this is a test file, please add license header

Reply:
Thank you for your review. 
Done!